src/components/Events.vue

```diff
@@ -2,7 +2,9 @@
 	<div>
 		<ul v-for="event in events" :key="event.key" class="event">
 			<li>{{ event.title }} {{ event.dateStart }}</li>
-			<h3 v-if="event.description">{{ event.description }}</h3>
+			<h3 v-if="event.description" v-linkified>
+				{{ event.description }}
+			</h3>
 			<p v-if="event.location">{{ event.location }}</p>
```

Comment:
I think it makes sense to have it on the description, but it should also be on the location, as I believe we're mainly going to use that for the meeting links.

Reply:
fixed in latest